ui/src/app/views/cohort-list/component.html

```diff
@@ -13,58 +13,9 @@ <h3 class="cohorts-header">Cohorts</h3>
       <span *ngIf="!writePermission" class="tooltip-content">Write permission required to create cohorts</span>
     </div>
   </div>
-  <div class="card item-card" *ngFor="let cohort of cohortList">
-    <div>
-      <div class="header-area">
-        <clr-dropdown [clrCloseMenuOnItemClick]="false">
-          <button type="button" [disabled]="actionsDisabled" clrDropdownTrigger>
-            <clr-icon shape="ellipsis-vertical" class="ellipsis-vertical" [class.disabled]="actionsDisabled"></clr-icon>
-          </button>
-          <clr-dropdown-menu *clrIfOpen>
-            <button class="action-item"
-                    [routerLink]="['build']"
-                    [queryParams]="{criteria: cohort.criteria}"
-                    [class.disabled]="actionsDisabled"
-                    [disabled]="actionsDisabled">
-              <clr-icon shape="copy"></clr-icon>
-              Clone
-            </button>
-            <button class="action-item edit-button"
-                    (click)="beginEdit(cohort)"
-                    [class.disabled]="actionsDisabled"
-                    [disabled]="actionsDisabled">
-              <clr-icon shape="pencil"></clr-icon>
-              Edit
-            </button>
-            <button class="action-item"
-                    routerLink="{{cohort.id}}/review"
-                    [class.disabled]="actionsDisabled"
-                    [disabled]="actionsDisabled">
-              <clr-icon shape="grid-view"></clr-icon>
-              Review
-            </button>
-            <button class="action-item delete-button"
-                    (click)="confirmDelete(cohort)"
-                    [class.disabled]="actionsDisabled"
-                    [disabled]="actionsDisabled">
-              <clr-icon shape="trash"></clr-icon>
-              Delete
-            </button>
-          </clr-dropdown-menu>
-        </clr-dropdown>
-        <div class="name"
-             [class.disabled]="actionsDisabled"
-             (click)="openCohort(cohort)">
-          {{cohort.name}}
-        </div>
-      </div>
-      <div class="description">{{cohort.description}}</div>
-    </div>
-    <div class="footer">
-      <div class="last-modified">Last Updated <br/>{{cohort.lastModifiedTime | date}}</div>
-      <div class="item-type cohort"><div>CH</div></div>
-    </div>
+  <div *ngFor="let cohort of resourceList">
+    <app-resource-card [resourceCard]="cohort" [cssClass]="'for-list'"
```

Comment:
To summarize offline Slack discussion, this parameter was to allow control for different styling on the homepage vs notebook list. I suggested to follow-up with Lou and verify these differences were intentional, since the card styling look pretty nearly identical to me in the latest version of the mocks. If they are meant to be the same, then we don't need this parameter.

If the styling is intentionally different, I would suggest switching to something more self-documenting, e.g. `[compactStyle]="true"`

Reply:
So the css I'm using matches what was already there. Thus needing the two different styles. The update to match current mocks is out of scope for this card. I can change the name.